samples/BoltOn.Samples.WebApi/Controllers/StudentsController.cs

```diff
@@ -1,46 +1,40 @@
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using BoltOn.Bus;
 using BoltOn.Mediator.Pipeline;
 using BoltOn.Samples.Application.DTOs;
 using BoltOn.Samples.Application.Handlers;
-using BoltOn.Samples.Application.Messages;
-using BoltOn.Samples.Domain.Entities;
 using Microsoft.AspNetCore.Mvc;
 
 namespace BoltOn.Samples.WebApi.Controllers
 {
-	[Route("api/[controller]")]
 	public class StudentsController : Controller
 	{
 		private readonly IMediator _mediator;
-		private readonly IBus _bus;
 
-		public StudentsController(IMediator mediator, IBus bus)
+		public StudentsController(IMediator mediator)
 		{
-			this._mediator = mediator;
-			this._bus = bus;
+			_mediator = mediator;
 		}
 
-		[HttpGet]
+		[HttpGet, Route("[controller]")]
```

Comment:
@gokulm : can't we add it on controller directly as its the same route ? anything m missing!

Reply:
@skankanvar does .net core still have routeprefix?

Reply:
@gokulm : not route prefix but route attribute at the controller itself ! but if m not wrong we need to put dummy Route("") at each action method if it doesn't have any other attribute other than specified at the controller route value.

Reply:
@skankanvar i'm not sure whether we can add route at the controller and at the action level. we used to add routeprefix at the controller level and route attribute at the action level